src/neuroglancer/sliceview/vector_graphics/vector_graphics_line_renderlayer.ts

```diff
@@ -0,0 +1,236 @@
+  }
+
+  endSlice(shader: ShaderProgram) {
+    let gl = this.gl;
+
+    gl.disableVertexAttribArray(shader.attribute('aVertexIndex'));
+    gl.disableVertexAttribArray(shader.attribute('aNormalDirection'));
+
+    gl.disableVertexAttribArray(shader.attribute('aVertexFirst'));
+    gl.disableVertexAttribArray(shader.attribute('aVertexSecond'));
+  }
+
+  draw(sliceView: SliceView) {
+    let visibleSources = sliceView.visibleLayers.get(this)!;
+    if (visibleSources.length === 0) {
+      return;
+    }
+
+    this.initializeShader();
+    if (this.shader === undefined) {
+      console.log('error: shader undefined');
+      return;
+    }
+
+    let gl = this.gl;
+
+    let shader = this.beginSlice(sliceView);
+
+    for (let _source of visibleSources) {
+      let source = _source as VectorGraphicsChunkSource;
+      let chunkLayout = source.spec.chunkLayout;
+      let chunks = source.chunks;
+
+      // Compute projection matrix that transforms vertex coordinates to device coordinates
+      gl.uniformMatrix4fv(
+          shader.uniform('uProjection'), false,
+          mat4.multiply(tempMat4, sliceView.dataToDevice, chunkLayout.transform));
+
+      let chunkDataSize: vec3|undefined;
+      let visibleChunks = sliceView.visibleChunks.get(chunkLayout);
+      if (!visibleChunks) {
+        continue;
+      }
+
+      for (let key of visibleChunks) {
+        let chunk = chunks.get(key);
+        if (chunk && chunk.state === ChunkState.GPU_MEMORY) {
+          let numInstances = chunk.numPoints / 2;  // Two points == One vector
+
+          this.vertexIndexBuffer.bindToVertexAttrib(
+              shader.attribute('aVertexIndex'),
+              /*components=*/2);
+
+          this.normalDirectionBuffer.bindToVertexAttrib(
+              shader.attribute('aNormalDirection'),
+              /*components=*/1);
+
+          const aVertexFirst = shader.attribute('aVertexFirst');
+          chunk.vertexBuffer.bindToVertexAttrib(
+              aVertexFirst,
+              /*components=*/3,
+              /*attributeType=*/GL_FLOAT,
+              /*normalized=*/false,
+              /*stride=*/6 * 4,
+              /*offset=*/0);
+          gl.ANGLE_instanced_arrays.vertexAttribDivisorANGLE(aVertexFirst, 1);
+
+          const aVertexSecond = shader.attribute('aVertexSecond');
+          chunk.vertexBuffer.bindToVertexAttrib(
+              aVertexSecond,
+              /*components=*/3,
+              /*attributeType=*/GL_FLOAT,
+              /*normalized=*/false,
+              /*stride=*/6 * 4,
+              /*offset=*/3 * 4);
+          gl.ANGLE_instanced_arrays.vertexAttribDivisorANGLE(aVertexSecond, 1);
+
+          gl.ANGLE_instanced_arrays.drawArraysInstancedANGLE(gl.TRIANGLE_STRIP, 0, 4, numInstances);
+
+          gl.ANGLE_instanced_arrays.vertexAttribDivisorANGLE(aVertexFirst, 0);
+          gl.ANGLE_instanced_arrays.vertexAttribDivisorANGLE(aVertexSecond, 0);
+        }
+      }
+    }
+    this.endSlice(shader);
+  }
+}
```

Comment:
This is very strange. I added a linebreak, but github doesn't seem to have taken notice... I'll see what I can do about that after the other changes are added to line_renderlayer.